backend/services/interfaces/statisticsService.ts

```diff
@@ -0,0 +1,36 @@
+import { SchoolResponseDTO } from "./schoolService";
+import { TestResponseDTO } from "./testService";
+
+export interface AggregateTestSessionDTO {
+  /** the ID of the corresponding test from the Test collection */
+  test: string;
+  /** the ID of the corresponding school from the School collection */
+  id: string;
+  /** the average overall test score */
+  averageScore: number;
+  /** the array of test result breakdowns */
+  resultsPerTestArr: number[][];
+  /** the array of average test scores for each Question */
+  averageScoresByQuestions: number[];
+}
+
+export interface StatisticsResponseDTO {
+  /** the corresponding test from the Test collection */
+  test: TestResponseDTO;
```

Comment:
Do we need to store `TestResponseDTO` for each `SchoolResponseDTO`? 
i.e maybe it makes more sense to store `school: Array<SchoolResponseDTO>` instead of `SchoolResponseDTO`? Let me know your thoughts here!
maybe then, it would also eliminate the need for `mapStatisticsToStatisticsDTOs` helper?